docs/sources/mimir/operators-guide/deploy-grafana-mimir/jsonnet/migrate-query-scheduler-from-dns-to-ring-based-service-discovery.md

```diff
@@ -0,0 +1,46 @@
+---
+description: Learn how to migrate query-scheduler from DNS-based to ring-based service discovery
+menuTitle: Migrate query-scheduler from DNS-based to ring-based service discovery
```

Comment:
Would `Migrate query-scheduler` drop too much context from the left-hand TOC?

Reply:
Unfortunately I think so. We can shorten it to "Migrate query-scheduler to ring-based service discovery"

Reply:
We agreed on Slack to edit it based on my suggestion.